packages/dioxus-blitz/src/lib.rs

```diff
@@ -93,17 +136,25 @@ fn launch_with_window<Doc: DocumentLike + 'static>(window: View<'static, Doc>) {
     let _guard = rt.enter();
 
     // Build an event loop for the application
-    let event_loop = EventLoop::<UserWindowEvent>::with_user_event()
-        .build()
-        .unwrap();
+    let mut builder = EventLoop::<UserWindowEvent>::with_user_event();
+
+    #[cfg(target_os = "android")]
+    {
+        use winit::platform::android::EventLoopBuilderExtAndroid;
+        builder.with_android_app(android_app);
+    }
+
+    let event_loop = builder.build().unwrap();
     let proxy = event_loop.create_proxy();
 
     // Multiwindow ftw
     let mut windows: HashMap<WindowId, window::View<'_, Doc>> = HashMap::new();
     let mut pending_windows = Vec::new();
 
     pending_windows.push(window);
-    let menu_channel = MenuEvent::receiver();
+
+    #[cfg(feature = "muda")]
```

Comment:
I reckon this feature should probably be called "menu" "menubar".

Reply:
Totally agree 👍 I went with the name `menu` in 823f309a566f45a231dc237b32ffa82349d419a0 but definitely open to `menubar` as well 

I also made the feature flag default, but not usable from Android/IOS, which I think could be easier for end users